### DISCUSSION TITLE
Enabling VS Code Snippets with LazyVim

### DISCUSSION THREAD

**Discussion by ashkan117** [LazyVim/LazyVim#2733] [Q&A]
Looking online I found a few things but trying the following code did not work for me. Running `LuaSnipListAvailable` shows all the maps as empty.

```lua
# plugins/snippets.lua
return {
  "rafamadriz/friendly-snippets",
  config = function()
    local vscode_path = "~/.config/Code/User/snippets"
    require("luasnip.loaders.from_vscode").lazy_load({ paths = { vscode_path } })
  end,
}

```

Side Question: Is it possible to use vscode snippets and Luasnippets at the same time?

**Comment by jyuan0** [comment]
I think you have to add that as a dependency of LuaSnip (it's in the [friendly-snippets docs](https://github.com/rafamadriz/friendly-snippets?tab=readme-ov-file#with-lazynvim)). See answer for: https://github.com/LazyVim/LazyVim/discussions/2253

> Side Question: Is it possible to use vscode snippets and Luasnippets at the same time?

Yes, just need to add another function call (see link above).

**Comment by surmish** [comment]
Is there a way to use `friendly-snippets` with native snippet support in Neovim? Using [this](https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/plugins/extras/coding/native_snippets.lua)
